x-pack/legacy/plugins/siem/server/lib/detection_engine/README.md

```diff
@@ -24,6 +24,13 @@ export KBN_URLBASE=http://${user}:${password}@localhost:5601
 
 source your .zhsrc/.bashrc or open a new terminal to ensure you get the new values set.
 
+Optional env var when set to true will utilize `reindex` api for reindexing
+instead of the scroll and bulk index combination.
+
+```
+export USE_REINDEX_API=USE_REINDEX_API
```

Comment:
Did you mean to use
```
export USE_REINDEX_API=true
```
here?

and then later use it like so:

```ts
process.env.USE_REINDEX_API === 'true'
```

Docs seem to suggest that?

Reply:
Yeah I wasn't sure what was more clear when I was working on it. I thought seeing `=== 'true'` looked kind of funny in code but I can re-write it to use the string 'true' instead of 'USE_REINDEX_API'.